lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -61,92 +62,96 @@ def _get_testable_interactive_backends():
     return backends
 
 
+_test_timeout = 10  # Empirically, 1s is not enough on Travis.
+
+
+# The source of this function gets extracted and run in another process, so it
```

Comment:
Can this become a docstring?

Reply:
That's not really a doc about the function but rather a comment about how it needs to be implemented?

Reply:
IMHO that's splitting hairs. But no strong opinion either way.